apps/contrib/templates/a4_candy_contrib/component_library.html

```diff
@@ -420,31 +417,152 @@ <h2 id="cl-alerts">Alerts</h2>
                         <button class="alert__close">X</button>
                     </p>
                 </div>
-
+                //Block Types
```

Comment:
This is showing in the component library. I think you can use html or django-template comments. Maybe the second are safer.